admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -159,7 +160,20 @@ private ReplicationField getReplicationFieldForConfig(ReplicatorConfig config) {
     field.destination(getSiteFieldForSite(siteManager.get(config.getDestination())));
     field.filter(config.getFilter());
     field.biDirectional(config.isBidirectional());
-    List<ReplicationStatus> statusList = history.getReplicationEvents(config.getName());
+    field.suspended(config.isSuspended());
+    field = setStatusRelatedFields(field, config);
+
+    return field;
+  }
+
+  private ReplicationField setStatusRelatedFields(ReplicationField field, ReplicatorConfig config) {
```

Comment:
❓ Why return field here? I would see the usefulness if above it was used for chaining but it is not.

Reply:
Imo it conveys more clearly that we are modifying the field. Even better would be to have the method on the field class so it would read field.setStatusRelatedFields() but then I would have to pass in the config, history, and replicator.

Reply:
ok. but then, from this method prototype, one would need to start using the returned field as you wouldn't know if a new one is returned or the same one.